src/python/WMCore/REST/Server.py

```diff
@@ -320,7 +320,7 @@ def static(self, *args, **kwargs):
 
         Serves assets as documented in :meth:`_serve`."""
         if len(args) > 1 or (args and args[0] != "yui"):
```

Comment:
On  0, ticoann <notifications@github.com> wrote:
> > @@ -320,7 +320,7 @@ def static(self, *args, **kwargs):
> >  
> >          Serves assets as documented in :meth:`_serve`."""
> >          if len(args) > 1 or (args and args[0] != "yui"):
> 
> I am not quite under stand this condition (although this is not what you changed). are we just supporting multiple file loading only when "yui" keyword is provided in the arguments?

yes, I think condition reads as if multiple arguments are provided then only
those with yui will be served. This is what I found "weird", but when code was
developed we probably only had YUI. For WMArchive I wanted to support
js/file.js, css/file.css, image/file.png etc. content. That's why I changed the
code.